From 2cb803f03ea87c27e20adacfbe2d692cd6dc3fab Mon Sep 17 00:00:00 2001
From: Jia <foo@example.com>
Date: Sat, 26 Mar 2022 16:10:14 +0800
Subject: [PATCH] Set user and password for sql connection

---
 appengine/monorail/framework/sql.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/appengine/monorail/framework/sql.py b/appengine/monorail/framework/sql.py
index 1d7573d56f..aecbef04f6 100644
--- a/appengine/monorail/framework/sql.py
+++ b/appengine/monorail/framework/sql.py
@@ -79,7 +79,7 @@ def cnxn_ctor(instance, database):
     if settings.local_mode:
       start_time = time.time()
       cnxn = MySQLdb.connect(
-        host='127.0.0.1', port=3306, db=database, user='root', charset='utf8')
+        host='127.0.0.1', port=3306, db=database, user='monorail', password='monorail', charset='utf8')
     else:
       start_time = time.time()
       cnxn = MySQLdb.connect(
-- 
2.25.1

